guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
+
+.Procedure
+. Disable Puppet server on {SmartProxies} before you disable it on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} plugin purge-puppet
+----
++
+If your deployment includes a database on {SmartProxy}, the command will also remove internal data from the Puppet database.
+. Disable Puppet server on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} plugin purge-puppet [--remove-all-data]
+----
++
+The command without arguments will also remove internal data from the Puppet database on {ProjectServer}.
+To remove external Puppet data as well, add the `--remove-all-data` argument to the command.
```

Comment:
This is a bit vague to me. Again because Puppet database makes me (as a Puppet user) think of PuppetDB.

@ezr-ondrej how can we phrase this correctly?

Reply:
Rephrased.